### PR TITLE
Add package-info to most classes

### PR DESCRIPTION
**Pull request by Ef55**
Note that not all packages are yet documented.

### CODE REVIEW

File: app/src/main/java/com/github/onedirection/startup/package-info.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/**
+ * Code called at application start up to initialize some functionality.
```
Comment (hugolan) [inline app/src/main/java/com/github/onedirection/startup/package-info.java:2]: could be more precise here

**Review by hugolan — APPROVED** [review]
good description of the classes

File: app/src/main/java/com/github/onedirection/startup/package-info.java (hunk 1 above)
Comment (Ef55) [reply app/src/main/java/com/github/onedirection/startup/package-info.java:2]: I personally don't have much more to say about this package.
If you have any suggesting, I would be more than happy to include them.

**Review by TheImpressiveDonut — APPROVED** [review]
Unbelievable documentation. It was so good that I fell from my chair !

**Pull request merged**